Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -23,7 +23,9 @@ final class PaymentViewController: UIViewController{
     }()
 
     private lazy var scrollView: UIScrollView = {
-        UIScrollView().disabledAutoresizingIntoConstraints()
+      let scrollView = UIScrollView().disabledAutoresizingIntoConstraints()
+      scrollView.keyboardDismissMode = .onDrag
```

Comment:
Not the biggest fan of this approach. The scroll view may not be immediately obvious and the scroll to dismiss also not very clear.

Previously I've approached this by using UIGestureRecogniser that would use a global function relying on `UIApplication.shared.resignFirstResponder()`.

I would be keen to get feedback on this if there's any external QA on real devices (don't ask for this, just observe any negative feedback @patrick-hoban-cko)

Reply:
it will be tested for full payment screen. other components are under development.